README.md

```diff
@@ -17,6 +17,12 @@ used.
 `pulp-ethernet` is intended for use with https://github.com/pulp-platform/ariane
 (a RISCV Linux-capable soft core).
 
+## Generate iDMA with AXIS support  (Terminal)
```

Comment:
Change to 

```
## Generate iDMA
```

since we generate it with all protocols

Reply:
i also updated the make rules in the readme as i can see they are modified in the mk